app/assets/javascripts/client/directives/learning_questions/learningQuestionModal.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('learningQuestionModal', learningQuestionModal);
+
+  function learningQuestionModal() {
+    return {
+      restrict: 'E',
+      scope: {
+        reminder: '@'
+      },
+      templateUrl: 'client/views/modals/learning_questions/add_learning_question.html',
+      link: function(scope) {
+        // The modal exists on the parent scope, so we have to reach up to it  to bind any closing actions on the modal.
+        var parentScope = scope.$parent;
```

Comment:
I don't think this is a problem at all, actually, in observer patterns is pretty normal to have multiple listeners, this is better than hardcoding the name of the variable in the parent scope, now `AssessmentDashboardSidebarCtrl`, `ResponseCtrl` and `SidebarResponseCardCtrl` would have a chance to freely rename it's modal variable at will without having to affect `learningQuestionModal`. you just gave `learningQuestionModal` one less reason to change, that's awesome.